.github/workflows/crowdin-contributors.yml

```
@@ -0,0 +1,41 @@
+name: Crowdin Contributors Action
+
+on:
+  schedule:
+    - cron: '0 0 * * 1' # https://crontab.guru/#0_0_*_*_1
+  # To manually run this workflow
+  workflow_dispatch:
+
+jobs:
+  crowdin-contributors:
+
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Generate Crowdin Contributors table
+      uses: andrii-bodnar/action-crowdin-contributors@v1.0.1
+      with:
+        contributors_per_line: 5
+        max_contributors: 10000
+        image_size: 100
+        min_words_contributed: 1
+        include_languages: true
+        placeholder_start: <!-- CROWDIN-CONTRIBUTORS-LIST:START -->
+        placeholder_end: <!-- CROWDIN-CONTRIBUTORS-LIST:END -->
+        crowdin_project_link: https://turingway.crowdin.com/turing-way
+
+      env:
+        CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
+        CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+        CROWDIN_ORGANIZATION: ${{ secrets.CROWDIN_ORGANIZATION }} # Optional. Only for Crowdin Enterprise
```

```suggestion
        CROWDIN_ORGANIZATION: ${{ secrets.CROWDIN_ORGANIZATION }} # Optional. Only for Crowdin Enterprise
```

If this is optional, why is it defined? Do we use Crowdin Enterprise? Have we added this secret to the repo?

---

For the comment, it was a mistake on my end, the CROWDIN_ORGANIZATION is not optional. I'll fix it asap

---

I can help with the token. I was actually thinking, it might be good to introduce [enviornments](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#environment-secrets) to this repo, so that each workflow only gets access to the secrets that it requires.

---

Can we delete the comment about it being optional as well please?